Mage/CommandCenter/TransportClient/CommandHTTPClient.cs

```diff
@@ -11,14 +11,12 @@ public class CommandHTTPClient : CommandTransportClient {
 	
 	//
 	private string endpoint;
-	private string username;
-	private string password;
+	Dictionary<string, string> headers;
```

Comment:
i know this is fine (functionally) but I really don't like the lack of "private" or "public" when reading code.